from distutils.log import error
from ssl import AlertDescription

try: 
    print( int(input())*2)
except:
    print("error")

#try-except文で，例外発生の処理
#https://atcoder.jp/contests/past201912-open/tasks/past201912_a